undefined_syms.txt

```diff
@@ -930,3 +930,8 @@ D_0601C160 = 0x0601C160;
 
 // z_bg_menkuri_eye
 D_06002D20 = 0x06002D20;
+
+// z_bg_mori_hashigo
+D_060037D8 = 0x060037D8;
+D_060036B0 = 0x060036B0;
+D_06004770 = 0x06004770;
```

Comment:
I noticed adding things to the end of undefined_syms.txt was more likely to cause merge conflicts so maybe avoid it (next times), I've been trying to follow alphabetical order but it's a mess anyway lol